Sources/Lang.php

```diff
@@ -255,6 +261,11 @@ public static function load(string $template_name, string $lang = '', bool $fata
 				break;
 			}
 
+			// Keep track of what we're up to, soldier.
+			if (!empty(Config::$db_show_debug)) {
+				Utils::$context['debug']['language_files'][] = $file[2] . '.' . $file[1] . ' (' . (Config::$languagesdir == $file[0] ? 'Base' : basename(Theme::$current->settings['theme_url'] ?? 'unknown')) . ')';
```

Comment:
```suggestion
				Utils::$context['debug']['language_files'][] = $file[2] . '/' . $file[1] . ' (' . (Config::$languagesdir == $file[0] ? 'Base' : basename(Theme::$current->settings['theme_url'] ?? 'unknown')) . ')';
```

I think that's what you meant here.